yarn.lock

```diff
@@ -1,16164 +0,0 @@
-# THIS IS AN AUTOGENERATED FILE. DO NOT EDIT THIS FILE DIRECTLY.
```

Comment:
I think something went wrong here 😅

Reply:
No idea how it got removed but readded now :D Not my day